tests/testcases/Group10-Vulnerability/10-05-Schedule-daily-scan.md

```diff
@@ -0,0 +1,25 @@
+Test 10-05  Schedule daily scan
+=======
+  
+# Purpose:  
+To verify daily scan work correctly  
+# References:  
+User guide  
+  
+# Environment:  
+* This test requires that a Harbor instance is running and available.  
+* Harbor is installed with clair enable.  
+* A linux host with Docker CLI installed.  
+  
+# Test Step  
+1. Login Harbor.  
+2. Push an image.  
+3. Schedule a daily scan.  
+4. Wait one day.  
+5. Check the timestamp of scan .  
+  
```

Comment:
should also test the case when the daily scan is disabled.  Maybe in a separate case and think about how to verify it quickly without having to wait for one day.